content/docs/04-clusters/03-edge/04-edgeforge-workflow/03.5-palette-canvos.md

```diff
@@ -0,0 +1,507 @@
+  <br />
+
+  ```bash
+  git clone https://github.com/spectrocloud/CanvOS.git
+  ```
+  
+
+2. Change to the **CanvOS/** directory. 
+  <br />
+
+  ```bash
+  cd CanvOS
+  ```
+
+
+3. The files relevant to the current guide are in the **v3.3.3** tag. Check out the desired tag. You can check out a more recent and stable version if available.
+  <br />
+
+  ```shell
+  git checkout v3.3.3
+  ```
+
+
+4. Review the relevant files.  
+  <br />
+
+  ```bash
+  .
+  ├── .arg          # Defines mutiple variables
+  ├── Dockerfile    # Bakes the arguments to use in the image
+  ├── Earthfile     # Series of commands for creating target images
+  ├── earthly.sh    # Script to invoke the Earthfile, and generate target images
+  └── user-data.template  # A sample user-data file
+  ```
+  <br />
+
+
+5. Edit the **.arg** file to customize the image tag for provider OS images, image registry, image repository, and OS distribution. 
+
+  The table below shows all arguments defined in the **.arg** file, their default, and possible values.
+
+  |**Argument**|**Description**|**Default Value**| **Other Possible Values** |
+  |---|---|---|---|
+  |`CUSTOM_TAG`|Tag for provider image|demo|Lowercase alphanumeric string without spaces|
+  |`IMAGE_REGISTRY`|Image registry name|ttl.sh|Your image registry URL, without http or https <br /> Example: hub.docker.com|
+  |`OS_DISTRIBUTION`|OS Distribution |ubuntu | opensuse-leap|
+  |`IMAGE_REPO`|Image repository name|`$OS_DISTRIBUTION`|Your image repository name|
+  |`OS_VERSION`|OS Version, only applies to Ubuntu |22| 20|
+  |`K8S_DISTRIBUTION`|Kubernetes Distribution |k3s| rke2, kubeadm |
+  |`ISO_NAME`|Name of the Installer ISO|palette-edge-installer|String without spaces|
+
+  Open up the **.arg** file in an editor and update the following variables. 
+  <br />
+
+    - Update the value for the `CUSTOM_TAG` variable to string made up of lowercase alphanumeric characters. This example uses `palette-learn` value. 
+    <br />
+
+    ```bash
+    CUSTOM_TAG=palette-learn
+    ```
+
+    - Update the value for the `IMAGE_REGISTRY` to point it to the Docker Hub host. If you are using any other image registry, the hostname must comply with standard DNS rules, but may not contain underscores.
+    <br />
+
+    ```bash
+    IMAGE_REGISTRY=registry.hub.docker.com
+    ```
+
+    - Update the value for the `OS_DISTRIBUTION` to OpenSuse Leap. This workflow supports OpenSuse Leap version 15.4.
+    <br />
+
+    ```bash
+    OS_DISTRIBUTION=opensuse-leap
+    ```
+
+    - Create a repository in the Docker Hub named `opensuse-leap` and set the repository's default visibility to public. Refer to [Create a repository](https://docs.docker.com/docker-hub/repos/create/#create-a-repository) instructions for creating a repository in the Docker Hub. The following declaration will default the repository name to be same as the OS distribtion. 
```

Comment:
This should be a prereq. So the interesting thing is that we now have different requirements. So what we can do is move the Prerequisites section inside the <Tabs> element and assign each one their own Prerequisites.